kong/clustering/services/sync/rpc.lua

```diff
@@ -88,7 +88,7 @@ function _M:init_cp(manager)
       sync_status = CLUSTERING_SYNC_STATUS.NORMAL,
       config_hash = fmt("%032d", default_namespace_version),
       rpc_capabilities = rpc_peers and rpc_peers[node_id] or {},
-    }, { ttl = purge_delay })
+    }, { ttl = purge_delay, no_broadcast_crud_event = true })
```

Comment:
Perhaps we should not have this `no_broadcast_crud_event` flag

Reply:
@chronolaw has rolled back this fix in this pr, we have a kag to track this.

Reply:
If it didn't cause issues. I would try to remove it from the start.

Reply:
I will add it in another, both full sync and incremental sync, do you think it is OK?